site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,6 +30,61 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
+
+
+### Chrome 91 (stable on May 25, 2021)
+
+Chrome 91 adds the following features to the extensions platform.
+
+#### ES modules for service workers
+
+JavaScript supports modules in service workers. Setting 'module' type by the constructor's type attribute, worker scripts are loaded as ES modules and the import statement is available on worker contexts. (https://www.chromestatus.com/guide/edit/4609574738853888)
```

Comment:
> Setting 'module' type by the constructor's type attribute, worker scripts are loaded as ES modules and the import statement is available on worker contexts.

But in extensions, the worker is automatically registered.  Instead, developers should specify it in their manifest:

```
"background": {
  "service_worker": "script.js",
  "type": "module"
}
```

Relatedly, we should update the relevant docs for that, and link to them here.

Reply:
@rdcronin, is this landing in 91? I hadn't heard an update yet about when it might land.
 
```suggestion
Chrome [is currently incubating][sw-module-feature] module support in service workers. As part of this work, we are also exploring exposign modules in extension service workers. See [background service workers](#TODO) for additional details.

[sw-module-feature]: [https://www.chromestatus.com/feature/4609574738853888]
```

Also Updates the link to point to the public version of the page. TBH, though, I'm not sure we should.

+1 to documenting how to use SW modules on another page & linking here. I think we can add a section to a page on background service workers with a warning aside that notes this feature is in development and can only be used in Canary or something.

Reply:
Use in extensions for the top-level SW is in M92.  I think module SWs on the web landed in M91, but I'd argue that that's not really something that fits into a "what's new in extensions" log

Reply:
Updated per review comments and created b/188936301 for the additional module docs work.